src/core/util/prefixer.js

```diff
@@ -0,0 +1,63 @@
+angular
+  .module('material.core')
+  .config( function($provide) {
+    $provide.decorator('$mdUtil', ['$delegate', function ($delegate) {
+
+      // Inject the prefixer into our original $mdUtil service.
+      $delegate.prefixer = MdPrefixer;

```

Comment:
Since most of the examples are prefixing for a querySelector, perhaps this would be better:

```js
$provide.decorator('$mdUtil', ['$delegate', function ($delegate) {
    var prefixer = new MdPrefix();
 
    $delegate.prefix = angular.bind(prefixer, prefixer.buildList);
    $delegate.prefixForQuery = angular.bind(prefixer, prefixer.buildSelector);
    $delegate.prefixer = prefixer;
}
```